src/main/java/org/kuali/kra/budget/document/BudgetDocumentRule.java

```diff
@@ -154,7 +154,122 @@ protected boolean processCustomSaveDocumentBusinessRules(Document document) {
     * @param budgetDocument
     * @return
     */
-    @SuppressWarnings("rawtypes")
+    @SuppressWarnings("rawtypes")  
```

Comment:
Is there anyway we could localize this suppression to the vars that necessitate it, or is that a large number?